File: The_Spy_Life/The_Spy_Life.java
import java.util.Scanner;

public class Program
{
public static String de(String str)
{
int strz = str.length();
String res ="";
for(int x =0;x<strz;x++)
{
int dx = strz-x-1;
if((str.charAt(dx) >= 'a' && str.charAt(dx) <= 'z') || (str.charAt(dx) >= 'A' && str.charAt(dx) <= 'Z') || str.charAt(dx) == ' ')
res += str.charAt(dx);
}
return res;
}
 public static void main(String[] args) 
{
  Scanner input = new Scanner(System.in);
  String msg = input.nextLine();
  System.out.print(de(msg));
}
}

File: The_Spy_Life/The_Spy_Life.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Sololearn
{
    class Program
    {
    public static string de(string str)
    {
        int strz = str.Length;
        char[] tmp = str.ToCharArray();
        Array.Reverse(tmp);
        
        string res = "";
        foreach(char x in tmp)
        {
        if((x >= 'a' && x <= 'z') || (x >= 'A' && x <= 'Z') || x == ' ')
            res += x;
        }
        
        return res;
    }
        static void Main(string[] args)
        {
     string msg = Console.ReadLine();
     Console.Write(de(msg));
        }
    }
}
